/projects/alar6830/software/anaconda/envs/BolT/lib/python3.7/site-packages/nilearn/__init__.py:67: FutureWarning: Python 3.7 support is deprecated and will be removed in release 0.12 of Nilearn. Consider switching to Python 3.9 or 3.10.
  _python_deprecation_warnings()
/projects/alar6830/software/anaconda/envs/BolT/lib/python3.7/site-packages/nilearn/input_data/__init__.py:23: FutureWarning: The import path 'nilearn.input_data' is deprecated in version 0.9. Importing from 'nilearn.input_data' will be possible at least until release 0.13.0. Please import from 'nilearn.maskers' instead.
  warnings.warn(message, FutureWarning)
Traceback (most recent call last):
  File "/projects/alar6830/software/anaconda/envs/BolT/lib/python3.7/site-packages/scipy/io/matlab/mio.py", line 39, in _open_file
    return open(file_like, mode), True
FileNotFoundError: [Errno 2] No such file or directory: '/projects/alar6830/BolT/Dataset/Prep/../fNIRS/HCP_MNI.mat'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "prep.py", line 24, in <module>
    prep(argv.atlas, argv.name, argv.fnirs )
  File "/projects/alar6830/BolT/Dataset/Prep/prep_abide.py", line 88, in prep_hcp
    fnirs_data, MNI_coords = load_fnirs(fnirs_folder)
  File "/projects/alar6830/BolT/Dataset/Prep/fnirs_utils.py", line 11, in load_fnirs
    digitization =scipy.io.loadmat(MNI_path)['MNI']
  File "/projects/alar6830/software/anaconda/envs/BolT/lib/python3.7/site-packages/scipy/io/matlab/mio.py", line 224, in loadmat
    with _open_file_context(file_name, appendmat) as f:
  File "/projects/alar6830/software/anaconda/envs/BolT/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/projects/alar6830/software/anaconda/envs/BolT/lib/python3.7/site-packages/scipy/io/matlab/mio.py", line 17, in _open_file_context
    f, opened = _open_file(file_like, appendmat, mode)
  File "/projects/alar6830/software/anaconda/envs/BolT/lib/python3.7/site-packages/scipy/io/matlab/mio.py", line 45, in _open_file
    return open(file_like, mode), True
FileNotFoundError: [Errno 2] No such file or directory: '/projects/alar6830/BolT/Dataset/Prep/../fNIRS/HCP_MNI.mat'
done prepping
